pkg/backup/processor.go

```diff
@@ -0,0 +1,561 @@
+/*
+Copyright 2017, 2019, 2020 the Velero contributors.
```

Comment:
should this just be 2020?

Reply:
maybe, I'm actually not sure - I copied it over as-is from `backup_controller.go` since most of the code in this file came from there but I don't know what the right policy is here.

Reply:
I "think" it being a new file, it should be 2020.  Only if it was a new file with the entirety of the code from the controller being moved (in essence, renaming and moving the entire file) it would make sense "to me".